js/index.js

```diff
@@ -0,0 +1,198 @@
+let playTurn = 1;
+let countSpaces = 0;
+let AI;
+
+
+const gameBoard = (space, value) => {
+  const board = [0, 0, 0, 0, 0, 0, 0, 0, 0];
+
+  const changeSpace = (space, value) => {
+    board[space] = value;
+  };
+  const getValue = (space) => (board[space]);
+
+  return { board, changeSpace, getValue };
+};
+
+const player = (name) => {
+  const victor = (player) => `${name} won!!!`;
+  const turn = (player) => (player.name);
+  return { name, victor, turn };
+};
+
+function init() {
+  const board = gameBoard();
+  return (board);
+}
+
+function initPlayers(name) {
+  const play = player(name);
+  play.victor();
+  return (play);
+}
+
+function updateSpace(board, space, value) {
+  board.changeSpace(space, value);
+}
+
+function render(board) {
```

Comment:
Let separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/